internal/docker/client.go

```diff
@@ -6,11 +6,11 @@ import (
 	"encoding/base64"
 	"encoding/json"
 	"fmt"
+	"github.com/docker/docker/api/types/build"
 	"io"
 	"os"
 	"time"
 
-	"github.com/docker/docker/api/types"
 	"github.com/docker/docker/api/types/image"
 	"github.com/docker/docker/api/types/registry"
 	"github.com/docker/docker/client"
```

Comment:
@jaredoconnell, do we care about the ordering of the imports?  (Ditto for `internal/docker/dockerClient.go` and `mocks/docker/dockerClient.go`.)